### DISCUSSION TITLE
Add item

### DISCUSSION THREAD

**Discussion by mrcsilverfox** [app-anvil/waste_not#61] [Q&A] [answered]
If a user has added an item and then he wants to add an item that is the same as an existing item (same expiration date, same storage, same measure), do we treat it as a different item?

**Comment by gianmarcocalbi** [comment, marked as answer]
> If a user has added an item and then he wants to add an item that is the same as an existing item (same expiration date, same storage, same measure), do we treat it as a different item?

While adding the item, if all inputs (name, expiration date, storage and measure) are the same we show an info box like "You have already the same item in your inventory, do you want to merge them?" + an additional button "Add to existing". We should still show the standard "Add" button if the user wants to add the item as a separate one.
